iidm/iidm-xml-converter/src/main/java/com/powsybl/iidm/xml/extensions/AbstractVersionableNetworkExtensionXmlSerializer.java

```diff
@@ -75,10 +75,14 @@ public String getNamespaceUri(String extensionVersion) {
 
     @Override
     public String getVersion() {
-        return getVersion(IidmXmlConstants.CURRENT_IIDM_XML_VERSION);
+        return getLastVersion(IidmXmlConstants.CURRENT_IIDM_XML_VERSION);
     }
 
-    public String getVersion(IidmXmlVersion networkVersion) {
+    public boolean versionExists(IidmXmlVersion networkVersion) {
+        return extensionVersions.containsKey(networkVersion);
+    }
+
+    public String getLastVersion(IidmXmlVersion networkVersion) {
```

Comment:
Not sure we should rename it because it's public, so strictly speaking it will be a breaking change.
Maybe it's better to just add some javadoc to better explain what it does?